plugins/modules/site_workflow_manager.py

```diff
@@ -857,7 +948,7 @@ def is_building_updated(self, updated_site, requested_site):
             It checks if the name, parent_name, latitude, longitude, and address (if provided) are
             equal, indicating that the building details have been updated. Returns True if the
             details match, and False otherwise.
-        """
+        # """
```

Comment:
How it works? Don't you get any error?

Reply:
This function checks for differences between the updated and requested site details to determine if changes are needed and it is called within site_requires_update, where errors are handled if an update is required

Reply:
You have added changes like 
```
  # """

My question is you added # before """ .. will it work?
```

Reply:
In description, By mistake added 
I removed